app/collins/models/shared/IpAddressable.scala

```diff
@@ -127,11 +127,11 @@ trait IpAddressStorage[T <: IpAddressable] extends Schema with AnormAdapter[T] w
   * For a range 0L..20L, used addresses List(17,18,19,20), the result will be None (allocate from beginning)
   */
   protected def getCurrentLowestLocalMaxAddress(calc: IpAddressCalc)(implicit scope: Option[String]): Option[Long] = inTransaction {
-    val minAddress = calc.minAddressAsLong
+    val startAddress = calc.startAddressAsLong
```

Comment:
I made https://github.com/tumblr/collins/issues/446 so that we (read I) remember to come back and fix this someday.